### PR TITLE
Add ability to add a table based on in-line YAML

### PR DESCRIPTION
**Pull request by nealkruis**

### CODE REVIEW

File: schema205/md/grid_table.py [hunk 1]
```diff
@@ -242,6 +242,7 @@ def string_out_table(dat, columns, caption, preferred_sizes=None, table_size="fo
     the_str = ""
     with io.StringIO() as handle:
         if table_size is not None:
+            handle.write("\n\\normalsize\n\n")
```
Comment (michael-okeefe) [inline schema205/md/grid_table.py:245]: I was surprised to see this here. It shouldn't cause any problems but is this a work-around for something else?

The table_size variable was meant to set the font-size stuff and return it back to normal (which, perhaps dangerously, we assume is "normalsize").

File: schema205/md/schema_table.py [hunk 2]
```diff
@@ -57,7 +57,8 @@ def data_elements_dict_from_data_groups(data_groups):
             new_obj = data_groups[dat_gr]["Data Elements"][element]
             new_obj["Name"] = f"`{element}`"
             if 'Required' in new_obj:
-                new_obj["Req"] = u'\N{check mark}' if new_obj["Required"] else ''
+                check = u'\N{check mark}'
```
Comment (michael-okeefe) [inline schema205/md/schema_table.py:60]: Just a note that I had to work around issues with the Unicode checkmark and the 205 docs. If you've found a way to get the checkmark to render cleanly into the PDF, then you're good. If not, you may want to check out the "enhancements-for-docs" branch and specifically https://github.com/open205/schema-205/blob/enhancements-for-docs/schema205/md/schema_table.py#L48

File: schema205/render_template.py [hunk 3]
```diff
@@ -199,7 +200,24 @@ def add_table(
                 target,
                 caption=caption,
                 add_training_ws=False)
-    return add_table
+    return add_schema_table
+
+def make_add_yaml_table():
+    """
+    RETURN:
+    """
```
Comment (michael-okeefe) [inline schema205/render_template.py:208]: Just a heads-up on the empty doc string but not a big deal.

**Review by michael-okeefe — APPROVED** [review]
Hi Neal, I think it looks good. I made a few comments for follow-up and FYI if desired but otherwise I don't see anything to hold this up.

File: schema205/md/schema_table.py (hunk 2 above)
Comment (nealkruis) [reply schema205/md/schema_table.py:60]: I figured out a way to handle the unicode. Thanks!

File: schema205/render_template.py (hunk 3 above)
Comment (nealkruis) [reply schema205/render_template.py:211]: Fixed in the latest commit.

File: schema205/md/grid_table.py (hunk 1 above)
Comment (nealkruis) [reply schema205/md/grid_table.py]: Good catch! This was an attempt to fix a problem where somehow these commands were affecting preceding section font size. It actually wasn't working and I found a better solution.

**Pull request merged**